src/auth/auth.go

```diff
@@ -148,6 +148,27 @@ func IsErrExpiredToken(err error) bool {
 	return strings.Contains(err.Error(), status.Convert(ErrExpiredToken).Message())
 }
 
+const errNoRoleBindingMsg = "no role binding exists for"
```

Comment:
Add a new type of error for when the specified role binding doesn't exist. This is necessary because we delete the output repo before we remove the pipeline from the input repo bindings.